pkg/types/types_test.go

```diff
@@ -75,6 +78,7 @@ func TestDecode(t *testing.T) {
 			GasOverheadPerUpkeep: 300_000,
 			MaxUpkeepBatchSize:   1,
 			ReportBlockLag:       0,
+			MercuryLookup:        false,
```

Comment:
not sure if already covered, but could you ensure that the missing value case is tested? (false being the default)

Reply:
i believe it's already covered in this test where all fields are missing and the expected default is false.
another similar test is the one above where only MercuryLookup is missing and expected value is false.